charts/castai-cluster-controller/templates/rbac.yaml

```diff
@@ -73,11 +73,11 @@ rules:
   # For installing/updating CAST AI components.
   - apiGroups: [ "rbac.authorization.k8s.io" ]
     resources: [ "roles" ]
-    resourceNames: [ "castai-cluster-controller", "castai-evictor", "castai-agent", "castai-spot-handler", "castai-kvisor", "castai-egressd" ]
+    resourceNames: [ "castai-cluster-controller", "castai-evictor", "castai-agent", "castai-spot-handler", "castai-kvisor", "castai-egressd", "castai-pod-pinner" ]
```

Comment:
you also need to extend this list https://github.com/castai/cluster-controller/blob/613c33dab377d57c1e0f4acbb42f25bfe1bcd455/helm/hook/hook.go#L36

Reply:
Updated: https://github.com/castai/cluster-controller/pull/89

We will also need to patch the `castai-pod-pinner` webhook, so I suppose we would need to add the below, correct?

```
  - apiGroups: [ "admissionregistration.k8s.io" ] #idk
    resourceNames: [ "castai-pod-pinner" ]
    resources: [ "mutatingwebhookconfigurations" ]
    verbs: [ "patch" ]
```

Reply:
Yep